app/webpacker/components/Panel/pages/MergeUsersPage/SpecialAccountDetails.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { List, Message } from 'semantic-ui-react';
+import _ from 'lodash';
+import { viewUrls } from '../../../../lib/requests/routes.js.erb';
+
+export default function SpecialAccountDetails({ user }) {
+  const specialAccountCompetitionKeys = _.keys(user.special_account_competitions);
+  if (!user.roles.length && !specialAccountCompetitionKeys.length) {
+    return null;
+  }
+
+  return (
+    <Message>
+      <Message.Header>{`The user with email ${user.email} is a special account.`}</Message.Header>
+      {user.roles.length > 0 && (
+        <p>
+          The user has some roles, you can view them
+          {' '}
+          <a href={viewUrls.users.rolesEditPage(user.id)}>here</a>
+          .
+        </p>
+      )}
+      <List divided relaxed>
+        {specialAccountCompetitionKeys.length && (
+        <List.Item>
+          {specialAccountCompetitionKeys.map((specialAccountCompetitionKey) => (
+            <List.Content>
+              <List.Header>{specialAccountCompetitionKey}</List.Header>
+              <List.Description>
+                {user.special_account_competitions[specialAccountCompetitionKey].join(', ')}
```

Comment:
Okay, the argument about the list growing too long for someone with many competitions is fair. Please add a comment here that you concsciously decided against a nested list / sub-list, so that in the future we don't accidentally "improve" this against WRT's will.